packages/domains-table/src/domains-table/domains-table-row.tsx

```diff
@@ -65,6 +75,13 @@ export function DomainsTableRow( {
 	const isManageableDomain = ! domain.wpcom_domain;
 	const shouldDisplayPrimaryDomainLabel = ! isAllSitesView && isPrimaryDomain;
 
+	const placeholderWidth = useMemo( () => {
+		const MIN = 40;
+		const MAX = 100;
+
+		return Math.floor( Math.random() * ( MAX - MIN + 1 ) ) + MIN;
+	}, [] );
```

Comment:
Interesting, this has me pondering what the "correct" way is to have a persistent value. I was thinking maybe `useRef`. But it probably doesn't make any difference.

Reply:
[Turns out we're not supposed to rely on `useMemo`](https://react.dev/reference/react/useMemo#caveats). I've changed it to `useState`.

Reply:
I just wanted to avoid recalculations through renders. I don't get why using `useMemo` is a bad idea in this case?

Reply:
If the width were to be recalculated each render then the width of the element would jump around. So any time a prop changed the user would see the placeholder width change. So it's not just to save on unnecessary/expensive calculations each render, it's a core part of the logic that the width of the placeholder shouldn't jump around.

The old React documentation makes it clearer, but the React team reserve the right to change `useMemo` in the future so that it actually recalculates sometimes (maybe to free up memory?) which means it should only be used for pure performance optimisations, not for things where the component would "break" if the value was recalculated.